libs/coin-evm/package.json

```diff
@@ -72,6 +72,7 @@
     "@types/invariant": "^2.2.2",
     "@types/jest": "^29.2.4",
     "@types/lodash": "^4.14.191",
+    "fast-check": "^3.12.0",
```

Comment:
Interesting lib, I don't find it very readable :sweat_smile: but why not.

Reply:
True. I like the idea of property based testing, can be useful in some cases. I guess we will see with time if we keep it or not (maybe overcomplexity / overengineering compared to benefits)